src/components/InputToDo.js

```diff
@@ -0,0 +1,39 @@
+import { useState } from 'react';
+import PropTypes from 'prop-types';
+// import { FaPlusCircle } from 'react-icons/fa';
```

Comment:
Kindly remove the commented line of code in your codes so that I will look clean

Reply:
Thanks for the suggestion, I will work on that.